low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -57,10 +58,19 @@ def process(self):
         if not os.path.exists(file_store_config_path):
             shutil.copyfile("%s/conf/sspl.conf.%s.yaml" % (SSPL_BASE_DIR, PRODUCT_NAME),
                             file_store_config_path)
-        if not os.path.exists(sample_global_config):
-            shutil.copyfile("%s/conf/sample_global_cortx_config.yaml" % (SSPL_BASE_DIR),
-                            sample_global_config)
-        Conf.load("sspl", "yaml://%s" % (file_store_config_path))
+
+        # Global config path in sspl.conf will be referred by sspl-ll service later.
+        Conf.load("sspl", sspl_config_path)
+        Conf.set("sspl", "SYSTEM_INFORMATION>global_config_url", self.global_config)
+        Conf.save("sspl")
+
+        # Update Rabbitmq values in sspl_tests.conf
+        Conf.load("sspl_test", sspl_test_config_path)
```

Comment:
Dont mix test code with main sspl service or setup code. Maintain it in sspl_test dir itself

Reply:
Here as well, Config overwrite steps for sspl_test. I will find a way to move this to sspl_test.

Reply:
Done.